experiments/LES/dycoms.jl

```diff
@@ -0,0 +1,176 @@
+    if xvert <= zi
+        θ_liq = FT(289)
+        q_tot = qref
+    else
+        θ_liq = FT(297.5) + (xvert - zi)^(FT(1/3))
+        q_tot = FT(1.5e-3)
+    end
+
+    # Calculate PhasePartition object for vertical domain extent
+    q_pt  = PhasePartition(q_tot, q_liq, q_ice)
+    Rm    = gas_constant_air(q_pt)
+
+    # Pressure
+    H     = Rm_sfc * T_sfc / grav;
+    p     = P_sfc * exp(-xvert/H);
+    # Density, Temperature
+    T = air_temperature_from_liquid_ice_pottemp_given_pressure(θ_liq, p, q_pt)
+    ρ = air_density(T, p, q_pt)
+
+    # Assign State Variables
+    u1, u2 = FT(6), FT(7)
+    v1, v2 = FT(-4.25), FT(-5.5)
+    w = FT(0)
+    if (xvert <= zi)
+        u, v = u1, v1
+    elseif (xvert >= ziplus)
+        u, v = u2, v2
+    else
+        m = (ziplus - zi)/(u2 - u1)
+        u = (xvert - zi)/m + u1
+
+        m = (ziplus - zi)/(v2 - v1)
+        v = (xvert - zi)/m + v1
+    end
+    e_kin       = FT(1/2) * (u^2 + v^2 + w^2)
+    e_pot       = grav * xvert
+    E           = ρ * total_energy(e_kin, e_pot, T, q_pt)
+
+    state.ρ     = ρ
+    state.ρu    = SVector(ρ*u, ρ*v, ρ*w)
+    state.ρe    = E
+    state.moisture.ρq_tot = ρ * q_tot
+
+    return nothing
+end
+
+function config_dycoms(FT, ArrayType, N, resolution, domain_size)
+    # Radiation model
+    κ             = FT(85)
+    α_z           = FT(1)
+    z_i           = FT(840)
+    ρ_i           = FT(1.13)
+    D_subsidence  = FT(3.75e-6)
+    F_0           = FT(70)
+    F_1           = FT(22)
+    radiation = StevensRadiation{FT}(κ, α_z, z_i, ρ_i, D_subsidence, F_0, F_1)
+
+    # Sources
+    # Rayleigh damping
+    zmax = domain_size[end]
+    zsponge = FT(1200.0)
+    rayleigh_sponge = RayleighSponge{FT}(zmax, zsponge, 1)
+    # Geostrophic forcing
+    f_coriolis    = FT(7.62e-5)
+    u_geostrophic = FT(7.0)
+    v_geostrophic = FT(-5.5)
+    geostrophic_forcing = GeostrophicForcing{FT}(f_coriolis, u_geostrophic, v_geostrophic)
+
+    # Boundary conditions
+    # SGS Filter constants
+    C_smag = FT(0.15)
+    C_drag = FT(0.0011)
+    LHF    = FT(115)
+    SHF    = FT(15)
+    bc = DYCOMS_BC{FT}(C_drag, LHF, SHF)
+
+    config = CLIMA.LES_Configuration("DYCOMS", FT, ArrayType, init_dycoms!,
+                                     radiation=radiation,
+                                     sources=(Gravity(),
+                                              rayleigh_sponge,
+                                              geostrophic_forcing),
+                                     bc=bc)
+    CLIMA.LES_SetupGrid!(config, N, resolution, domain_size)
+
+    return config
+end
+
+function main()
+    CLIMA.init()
+
+    FT = Float32
+
+    # DG polynomial order
+    N = 4
+
+    # Domain resolution and size
+    Δh = FT(40)
+    Δv = FT(10) #Δh/aspectratio
+    aspectratio = Δh/Δv
+    resolution = (Δh, Δh, Δv)
+
+    xmin, xmax = 0, 5000
+    ymin, ymax = 0, 5000
+    zmin, zmax = 0, 2500
+    domain_size = (xmin, xmax, ymin, ymax, zmin, zmax)
+
+    dt = FT(0.01)
```

Comment:
We should have a similar construction here as above, with something like
`dt = Courant_number * min_node_distance(config.grid, HorizontalDirection()) / acoustic_speed`